src/ergw_sx_node.erl

```diff
@@ -450,7 +450,7 @@ handle_event({call, From},
 	     _State,
 	     #data{pfcp_ctx = #pfcp_ctx{seid = #seid{dp = SEID}} = PCtx, tdf = Tdf}) ->
     {ok, {tdf, VRF}} = ergw_pfcp:find_urr_by_id(Id, PCtx),
-    ?LOG(debug, "Sx Node TDF Report on ~p for UE IPv4 ~p IPv6 ~p", [VRF, IP4, IP6]),
+    ?LOG(debug, "Sx Node TDF Report on ~p for UE IPv4 ~p IPv6 ~p", [VRF, ergw_inet:bin2ip(IP4), ergw_inet:bin2ip(IP6)]),
```

Comment:
If you insist on formating IPs for debug logging, then format to something human readable and not from binary to Erlang tuples.

Reply:
I was think that `{x, x, x, x}` will be normal. What is your preferred format?

Reply:
`inet:ntoa/1`

Reply:
Done